### PR TITLE
Added loading gif for better visualisation

### PR DESCRIPTION
**Pull request by jarvisdev**
solved issue #78

### CODE REVIEW

File: Projects/templates/Projects/home.html [hunk 1]
```diff
@@ -185,8 +185,14 @@ <h4>
         var infiniteScroll = new Waypoint.Infinite({
           element: $('.issues-container')[0],
           items: '.issue-list-item',
-          loadingClass: 'more-issues-loading',
-          more: '.more-issues-link'
+          //loadingClass: 'more-issues-loading',
```
Comment (dbads) [inline Projects/templates/Projects/home.html]: Cool

File: Projects/templates/Projects/home.html [hunk 2]
```diff
@@ -103,10 +103,10 @@ <h4>
     {% endif %}
   </div>
 
-<!--  <div class="more-issues-loading container">
-      <img src="static/Projects/img/loading.gif" alt="loading more issues" width="100%">
+ <div class="more-issues-loading container">
```
Comment (dbads) [inline Projects/templates/Projects/home.html]: Make sure.. this gif doesn't fit to the place and doesn't look awkward :smile:

**Comment by dbads** [comment]
You need to create a request for verification for achieving points for this PR :smile:

**Comment by jarvisdev** [comment]
but sir on contrihub-18 website there is no card made for this issue. I cannot find any to request to merge my pull request.

**Pull request merged**